### PR TITLE
To Do List basic structure

### PR DESCRIPTION
**Pull request by edenlisk**
In this PR I did the following:
1. Implemented to do list basic structure
2. Populated the list with sample information

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "webpack",
+    "start": "webpack --config webpack.config.js",
```
Comment (ha-manel) [inline package.json]: - [x] Kindly update the scripts and replace them with this so we can start the server and build the project properly 👍 

``` 
  "test": "echo \"Error: no test specified\" && exit 1",
  "start": "webpack serve --open",
  "build": "webpack"
```

File: src/template.html [hunk 2]
```diff
@@ -7,7 +7,11 @@
 </head>
 <body>
     <div>
-        <ul></ul>
+        <p>Today's To Do</p>
+        <span class="reload-icon"></span>
+    </div>
+    <div class="todo-container">
+        <ul class="todo-list list-unstyled"></ul>
```
Comment (ha-manel) [inline src/template.html]: - [x] Kindly add the input field and please try to style the app to match the provided template 💯😄 

image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @edenlisk 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Professional README file ✔️

## Required Changes:

- [ ] Kindly try to follow best practices when you write commit messages, they should be capitalized sentences in the imperative form such as: "Add linters" instead of "linters added" ...etc. 👍 

_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,112 @@
...
+
+Feel free to check the [issues page](https://github.com/edenlisk/ToDoList/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please consider giving a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (elmar8287) [inline README.md:112]: - [ ] OPTIONAL I suggest you add the section with a link to your license. So, here is the link `https://github.com/edenlisk/ToDoList/blob/todolist_basic_structure/LICENSE`

**Review by elmar8287 — APPROVED** [review]
Hi, @edenlisk.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- All suggestions from the previous reviewer are implemented 👍 

### Optional suggestions

- [ ] Please, in the future, consider giving the titles for commits with the Capital letter, to make your commits history looks professional 👍 
- [ ] Please, find my optional suggestion under the review 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**